Cannot build for Varnish 4.1 using branch 4.1

The `4.1` branch requires varnish 5.0+ which causes `./configure` to fail.

```
checking for Varnish... 4.1.11
configure: error: Varnish version 5.0 or higher is required.
```

Upon changing the `VARNISH_PREREQ([5.0])` constraint in `configure.ac` it has been built successfully. Is it only the matter of a wrong constraint?

--- a/configure.ac
+++ b/configure.ac
@@ -28,7 +28,7 @@ AC_ARG_WITH([rst2man],
 
 m4_ifndef([VARNISH_PREREQ], AC_MSG_ERROR([Need varnish.m4 -- see README.rst]))
 
-VARNISH_PREREQ([5.0])
+VARNISH_PREREQ([4.1])
 VARNISH_VMODS([accept])
 
 VMOD_TESTS="$(cd $srcdir/src && echo tests/*.vtc)"

